fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MatcherHelper.kt

```diff
@@ -75,8 +80,10 @@ object MatcherHelper {
         for (appItem in appItems) {
             if (exists) {
                 assertTrue(appItem.waitForExists(waitingTime))
+                Log.i("MozTestLog", "assertItemContainingTextExists: Verified ${appItem.selector} exists")
             } else {
                 assertFalse(appItem.waitForExists(waitingTimeShort))
+                Log.i("MozTestLog", "assertItemContainingTextExists: Verified ${appItem.selector} does not exist")
```

Comment:
Can we declare the tag in the Class, as well as use `UI` in the tag?

e.g, private static final String TAG = "MozUITestLog";

Reply:
Thanks @AaronMT , ss discussed I've added it to [Constants](https://github.com/mozilla-mobile/firefox-android/pull/4390/files#diff-a219bf2265cd9fa8230ab386bf0e0a7a262fdcd3984dbccbf581d659e8f62efbR12)